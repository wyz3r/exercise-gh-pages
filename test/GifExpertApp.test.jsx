import { fireEvent, render, screen } from "@testing-library/react"
import { GifExpertApp } from "../src/GifExpertApp"




describe('Testing component <GifExpertApp />', () => { 

    test('should match initial values snapshot', () => { 

        const {container} =  render(<GifExpertApp />);
        expect(container).toMatchSnapshot()
        const input = screen.getByRole('textbox')
        const form = screen.getByRole('form')
        
        fireEvent.input(input, {target:{value: 'Charizard'}})
        fireEvent.submit(form)
        

     })
 })