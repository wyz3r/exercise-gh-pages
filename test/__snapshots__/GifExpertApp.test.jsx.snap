// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component <GifExpertApp /> should match initial values snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Search Gif"
      type="text"
      value=""
    />
  </form>
  <h3 />
  <h2>
    Loading...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
